app/javascript/src/Types/libs/libdefs.js

```diff
@@ -9,7 +9,7 @@ declare var module: {
 
 // TODO: remove these module declarations when not failing
 declare module 'whatwg-fetch' {
-
+  declare module.exports: any
```

Comment:
Let's reduce the use of any as much as possible
```suggestion
  declare module.exports: fetch
```